TicTacToe.cpp

```diff
@@ -128,22 +128,39 @@ int TicTacToe::checkVictoryVertical(int teamSide) // check all (9) vertical colu
 
 void TicTacToe::viewAll()
 {
-	// Prints out the field with relevant team integers
-	std::cout << "    " << field[2][0][2] << " --- " << field[2][1][2] << " --- " << field[2][2][2] << std::endl;
+	int hold[26];   //create array to hold field integers
```

Comment:
When declaring the size of an array, do not use 0-indexing. If it needs to hold 27 integers, it should be hold[27].